core/snapshot/params.go

```diff
@@ -33,6 +37,8 @@ var params = &node.PluginParams{
 		"nodeConfig": func() *flag.FlagSet {
 			fs := flag.NewFlagSet("", flag.ContinueOnError)
 			fs.Bool(CfgPruningEnabled, true, "whether to delete old message data from the database")
+			fs.String(CfgPruningTargetDatabaseSize, "35GB", "target size of the database (0 = disable pruning by database size)")
```

Comment:
Move this below "CfgPruningDelay" to match the order in the constants and the config